managedApplication/parameters-template.json

```diff
@@ -4,5 +4,6 @@
     "dbServerType":         { "value": "mysql" },
     "fileServerType":       { "value": "nfs" },
     "autoscaleVmSku":       { "value": "Standard_DS1_v2" },
-    "fileServerDiskCount":  { "value": 2 }
+    "fileServerDiskCount":  { "value": 2 },
+    "searchType":           { "value": "azure-search" }
```

Comment:
@rgardler - Do you want azure-search to be on by default on managed application? I don't think so, but confirming with you.